dali/pipeline/executor/executor.h

```diff
@@ -207,7 +207,13 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
       }
   }
 
-  void HandleError(const char *message = "Unknown exception") {
+  void HandleError(const std::string &stage, const std::string &op_name,
+                   const std::string &instance_name, const std::string &message) {
+    HandleError(make_string("Error when executing ", stage, " Operator ", op_name,
```

Comment:
```suggestion
    HandleError(make_string("Error when executing ", stage, " operator ", op_name,
```

Reply:
1. Trim leading underscores from op_name, if there are any (ExternalSource and TFRecordReader use them).
2. Some operators already add their name to error messages - sometimes going to great lengths to get it correct (Reshape, Reinterpret, Warp operator family). Sometimes there's kernel name added, which may coincide with operator name - in either of these cases (especially kernels) it would be nice to also trim ```"<op_name>: "```

```c++
if (op_name[0] == '_') op_name = op_name.substr(1);  // trim leading underscore
if (message.rfind(op_name + ": ", 0) == 0)  // trim "<op_name>: "
  message = message.substr(op_name.length() + 2);
```

Reply:
1. Done
2. Done - still, it won't work with "KernelCpu: <error msg>"

Reply:
And done Operator -> operator.